docs/ReleaseHistory.md

```diff
@@ -11,7 +11,17 @@
 - FPS => False positive reduction in static analysis.
 - FNS => False negative reduction in static analysis.
 
-# 1.16.0 - 03/05/2025
+# UNRELEASED
+- BRK: `RegexPattern.Pattern` is no longer virtual.
+- BRK: `RegexPattern.Signatures` is no longer virtual.
+- BRK: `IdentifiableKey.ChecksumSeeds` is no longer virtual.
+- BRK: `RegexPattern.RotationPeriod` is no longer publicly settable.
+- BRK: `IdentifiableKey.RegexNormalizedSignature` is removed.
+- BRK: Abstract classes `IdentifiableKey`, `Azure32ByteIdentifiableKey`, `Azure64ByteIdentifiableKey`, and `AzureMessagingIdentifiableKey` now require derived classes to pass their signature to the base constructor.
+- PRF: Remove unnecessary and expensive recomputation of `RegexPatter.Pattern`, `RegexPattern.Signatures`, and `IdentifiableKey.ChecksumSeeds` on every property access.
+- NEW: Sort properties by name in GeneratedRegexPatterns/*.json.
```

Comment:
>NEW [](http://example.com/codeflow?start=2&length=3)

NIT: `NEW` labeled items should appear before `PRF`. we group and order release note entries following the ordering in the `Definitions` legend.

Reply:
Fixed.